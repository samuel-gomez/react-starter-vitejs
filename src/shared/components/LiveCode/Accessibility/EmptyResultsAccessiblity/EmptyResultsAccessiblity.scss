@import '@axa-fr/react-toolkit-core/src/common/scss/core.scss';

.empty-accessiblity {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 300px;
  color: $color-texte;
  font-style: italic;
  font-weight: 500;
}
