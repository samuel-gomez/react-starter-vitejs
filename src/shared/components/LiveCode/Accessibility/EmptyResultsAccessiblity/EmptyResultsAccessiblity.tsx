import './EmptyResultsAccessiblity.scss';

const EmptyResultsAccessiblity = ({ message }: { message: string }) => <div className="empty-accessiblity">{message}</div>;

export default EmptyResultsAccessiblity;
