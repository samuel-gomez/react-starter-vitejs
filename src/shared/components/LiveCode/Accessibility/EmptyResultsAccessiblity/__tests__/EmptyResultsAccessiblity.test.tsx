import { render } from '@testing-library/react';
import EmptyResultsAccessiblity from '../EmptyResultsAccessiblity';

describe('<EmptyResultsAccessiblity />', () => {
  it('Render <EmptyResultsAccessiblity /> with message', () => {
    const { asFragment } = render(<EmptyResultsAccessiblity message="test" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
