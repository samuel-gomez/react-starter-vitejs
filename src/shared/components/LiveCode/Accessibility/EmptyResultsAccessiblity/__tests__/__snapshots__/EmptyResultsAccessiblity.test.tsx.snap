// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<EmptyResultsAccessiblity /> > Render <EmptyResultsAccessiblity /> with message 1`] = `
<DocumentFragment>
  <div
    class="empty-accessiblity"
  >
    test
  </div>
</DocumentFragment>
`;
